File: coding-challenges/week09/day1/stack.py
def perfect(string):
    stack = []
    dict = { ")":"(" , "}":"{","]":"["}

    for i in string:
        


        if i in dict:
            if stack == [] or dict[i] != stack.pop():
                return False
            
            
        
        else:
            stack.append(i)
            
            

    
    return stack == []

    # else: 
    #     False

print (perfect("(([]))"))

    

File: coding-challenges/week26/day03/stack.js
let push = document.querySelector('#push')
let pop = document.querySelector('#pop')
let stack = [];
console.log(push)

function addtostack (){
    let item = document.getElementById('number').value;
    console.log(item)
    stack.push(item)
    document.getElementById('result').innerHTML = stack;
}


document.addEventListener('DOMContentLoaded',function(){
    push.addEventListener('click',addtostack)

});
pop.addEventListener('click', function(){
    if (stack != []) stack.pop();
    document.getElementById('result').innerHTML = stack;
    
})


